tidy3d/web/core/environment.py

```diff
@@ -164,5 +165,16 @@ def set_current(self, config: EnvironmentConfig) -> None:
         """
         self._current = config
 
+    def enable_caching(self, enable_caching: bool = True) -> None:
+        """Set the current environment.
```

Comment:
this should probably be "Set the environment configuration setting with regards to caching simulation results."